scripts/validate-test-results

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env perl
+
+# Copyright 2023 SUSE LLC
```

Comment:
It's not clear to me if this file (and others) are new custom files you create for this distri or if you get them from some draft branch in os-autoinst-common. I suggest to make that clear.

Specifically here please use

```suggestion
# Copyright SUSE LLC
```